crates/c-api/src/store.rs

```diff
@@ -60,16 +76,17 @@ pub extern "C" fn wasm_store_new(engine: &wasm_engine_t) -> Box<wasm_store_t> {
 /// foreign data and configuring WASI as well.
 #[repr(C)]
 pub struct wasmtime_store_t {
-    pub(crate) store: Store<StoreData>,
+    pub(crate) store: Store<WasmtimeStoreData>,
 }
 
 wasmtime_c_api_macros::declare_own!(wasmtime_store_t);
 
-pub type CStoreContext<'a> = StoreContext<'a, StoreData>;
-pub type CStoreContextMut<'a> = StoreContextMut<'a, StoreData>;
+pub type WasmtimeStoreContext<'a> = StoreContext<'a, WasmtimeStoreData>;
+pub type WasmtimeStoreContextMut<'a> = StoreContextMut<'a, WasmtimeStoreData>;
 
```

Comment:
Could these renamings perhaps be split out either commit-wise or PR-wise? It's a bit difficult to disentangle what's going on here with simultaneous renamings.

Reply:
Split out the renaming to https://github.com/bytecodealliance/wasmtime/pull/8344